integrations/cdi/eclipselink-cdi/src/main/java/io/helidon/integrations/cdi/eclipselink/CDISEPlatform.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Usual practice is to remove "All rights reserved." from updated copyright notices.

Reply:
Fixed in https://github.com/oracle/helidon/pull/2263/commits/8086f86bc49c7fa3989a1f29eeab2d3df42e8dca.